docs: misc academy docs fixes (#2536)
apify/apify-docs: sources/academy/platform/expert_scraping_with_apify/managing_source_code.md

Then, we'll run the commands it tells us in our terminal (while within the **demo-actor** directory) to initialize the repository locally, and then push all of the files to the remote one.

> After you've created your repo, navigate on the Apify platform to your Amazon scraping Actor (referred to throughout this course as **demo-actor**). In the **Source** tab, click the dropdown menu under **Source code** and select **Git repository**. By default, this is set to **Web IDE**, which is what we've been using so far.

Reviewer: ```suggestion
After you've created your repo, in the Apify platform navigate to your Amazon scraping Actor that we called **demo-actor**. In the **Source** tab, click the dropdown menu under **Source code** and select **Git repository**. By default, this is set to **Web IDE**, which is what we've been using so far.
```
Let's avoid brackets.

![Select source code location](./images/select-source-location.png)

Then, go ahead and paste the link to your repository into the **Git URL** text field and click **Save**.

> The final step is to check your **Build settings**. Under the Git URL field, you'll see two options: **Automatic builds** and **Manual builds**. Make sure **Automatic builds** is selected - this tells Apify to rebuild your Actor whenever you push to GitHub, with no extra configuration needed.

Reviewer: ```suggestion
The final step is to check your **Build settings**. Under the Git URL field, there're two options: **Automatic builds** and **Manual builds**. Select **Automatic builds**. It tells Apify to rebuild your Actor whenever you push to GitHub, with no extra configuration needed.
```
